sdk/python/kfp/dsl/_container_op.py

```diff
@@ -630,6 +630,95 @@ def inputs(self):
         return _pipeline_param.extract_pipelineparams_from_any(self)
 
 
+class InitContainer(Container):
```

Comment:
should we update doc [here](https://www.kubeflow.org/docs/pipelines/sdk/sdk-overview/) to add initcontainer?
do we also need a sample to illustrate how to use initcontainer?

Reply:
yeah i'll add it separately.